website/pages/submission/tasks/run_analysis.mdx

```diff
@@ -210,21 +210,36 @@ Description of the variables:
 #### Final steps
 
 - Rename the `env.example` to `.env` and save the file with all the required variables updated.
+
 - In this demo, the analysis is running with the query (named as `user_query`) which is located in the end of the `analysis_engine.py` file:
 
 ```sql
-SELECT value_as_number FROM public.measurement
+query_template = Template("""SELECT value_as_number FROM $schema
 WHERE measurement_concept_id = 21490742
-AND value_as_number IS NOT NULL
+AND value_as_number IS NOT NULL""")
+
+# Automatically uses your environment variables
+sql_schema = os.getenv("SQL_SCHEMA", "public.measurement")
+
+user_query = query_template.safe_substitute(
+    schema=sql_schema
+)
 ```
 
-This assumes that the OMOP CDM is under the public schema. If it is under a different schema, you need to change the `public.measurement` to the actual schema name.
-For example, if the schema name of OMOP CDM is `testing`, the query should be:
+**Configuring Database Schema** 
 
-```sql
-SELECT value_as_number FROM testing.measurement
-WHERE measurement_concept_id = 21490742
-AND value_as_number IS NOT NULL
+The analysis engine now supports configurable database schemas through environment variables. Add these to your .env file:
+
+```bash 
+SQL_SCHEMA=public.measurement
+```
+
+**Running the Analysis**
+
+This assumes that the OMOP CDM is under the public schema. If it is under a different schema, you need to change the `public.measurement` to the actual schema name in the .env file. For example, if the schema name of OMOP CDM is `testing`, the .env variable should be:
```

Comment:
Just to note let's get the changes correct in https://github.com/Health-Informatics-UoN/Five-Safes-TES-Analytics/pull/30

And then update here